vsock_proxy/service/nitro-enclaves-vsock-proxy.service

```diff
@@ -8,7 +8,8 @@ Type=simple
 StandardOutput=journal
 StandardError=journal
 SyslogIdentifier=vsock-proxy
-ExecStart=/bin/bash -ce "REGION=$(curl --silent http://169.254.169.254/latest/dynamic/instance-identity/document | jq -r .region) ; \
+ExecStart=/bin/bash -ce "TOKEN=$(curl --silent -X PUT \"http://169.254.169.254/latest/api/token\" -H \"X-aws-ec2-metadata-token-ttl-seconds: 21600\") ; \
```

Comment:
Should we consider a fallback to IMDSv1 in case v2 isn't available?

Reply:
I was thinking of this, just that the request mentioned to update the "code to only use IMDSv2". We may need to require some exception / share additional info to keep also IMDSv1.